src/main/kotlin/app/cash/licensee/pluginExtension.kt

```diff
@@ -82,6 +88,11 @@ interface LicenseeExtension {
     options: Action<AllowDependencyOptions> = Action { },
   )
 
+  fun allowDependency(
+    dependencyProvider: Provider<out Dependency>,
```

Comment:
We could limit the type parameter from `Dependency` to `ExternalModuleDependency` because license information are only available for external deps, I guess (?), but on the other hand, I don't see a use-case for limiting the type parameter.